### DISCUSSION TITLE
About train datasets

### DISCUSSION THREAD

**Discussion by jianjiandandande** [STVIR/pysot#442] [Reproduce] [answered]
Different frames range in different data sets, what do they represent? And when train the SiamRPN++, I found that with three data sets (COCO, DET, VID) and with four data sets (COCO, DET, VID, YOUTUBE BB), the total number in the log seems to be the same

**Comment by jianjiandandande** [comment]
![image](https://user-images.githubusercontent.com/15951818/95038734-76893f80-0701-11eb-8c47-0bc04a4708fa.png)

![image](https://user-images.githubusercontent.com/15951818/95038779-9a4c8580-0701-11eb-9fc5-86cff1964a2a.png)

**Reply by ssbagalkar** [reply]
I have the same question!

**Comment by ZhiyuanChen** [comment, marked as answer]
`FRAME_RANGE` describes the distance between search and template
`NUM_USE` describes the amount of data used
